src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -92,13 +160,19 @@ public function configure(array $configuration): void
     {
         parent::configure($configuration);
 
+        $simpleTags = array_intersect($this->configuration['tags'], self::SIMPLE_TAGS);
         $tagsWithNameToAlign = array_intersect($this->configuration['tags'], self::TAGS_WITH_NAME);
         $tagsWithMethodSignatureToAlign = array_intersect($this->configuration['tags'], self::TAGS_WITH_METHOD_SIGNATURE);
-        $tagsWithoutNameToAlign = array_diff($this->configuration['tags'], $tagsWithNameToAlign, $tagsWithMethodSignatureToAlign);
+        $tagsWithoutNameToAlign = array_diff($this->configuration['tags'], $simpleTags, $tagsWithNameToAlign, $tagsWithMethodSignatureToAlign);
         $types = [];
 
         $indent = '(?P<indent>(?:\ {2}|\t)*)';
 
+        // e.g. @deprecated
+        if ([] !== $simpleTags) {
+            $types[] = '(?P<tag>'.implode('|', $simpleTags).')';
+        }
+
         // e.g. @param <hint> <$var>
         if ([] !== $tagsWithNameToAlign) {
             $types[] = '(?P<tag>'.implode('|', $tagsWithNameToAlign).')\s+(?P<hint>(?:'.TypeExpression::REGEX_TYPES.')?)\s+(?P<var>(?:&|\.{3})?\$\S+)';
```

Comment:
> I don't understand why the tests fail. 

similar sub-pattern `<tag>`

Reply:
+ `(?P<var>(&|\.{3})\s*\$[\w]+\S+)`

Reply:
if `<hint>` not required you must use `(?P<hint>('.TypeExpression::REGEX_TYPES.')\s+)*?`

Reply:
I'm not sure I understand what you mean.

Reply:
@niklam you add sub-pattern `tag` in line 173, but sub-pattern with same name already defined in line 178, pattern name or combine them

Reply:
Oh, right. Thanks. I renamed the newly added sub-pattern to `simpleTag`, and added processing for it to 432.